test/types/range/stonea/rangeKeyword.good

```diff
@@ -0,0 +1,5 @@
+rangeKeyword.chpl:2: error: attempt to redefine reserved type 'range'
+rangeKeyword.chpl:3: error: attempt to redefine reserved type 'range'
+rangeKeyword.chpl:4: error: attempt to redefine reserved type 'range'
+rangeKeyword.chpl:5: error: attempt to redefine reserved type 'range'
```

Comment:
What happens to a user who was relying on this variable name? Was there a discussion on if this should be a warning to provide a grace period to users to change over?

Although I suppose we could argue this is a bug fix

Reply:
Good question.

I had some conversation about the same issue in regards to `init`/`deinit`/`postinit` here:

https://cray.slack.com/archives/GDP57GWNS/p1693351953294249

And at least from a few key folks I got the "go ahead" for deprecating reserved words without going through the normal cycle.

Reply:
That works for me!